g2p/tests/public/data/eng.csv

```diff
@@ -1,3 +1,5 @@
 # Make sure the same input in NFD and NFC both get handled correctly
 eng-ipa,eng-arpabet,ĩ,IY N
 eng-ipa,eng-arpabet,ĩ,IY N
+eng,eng-ipa,hello,hʌloʊ
+eng,eng-arpabet,hello,HH AH L OW
```

Comment:
Yes, I'm not entirely sure what the best answer is here, possibly making tokenization implicit with lexicon mappings, because in some cases they may need to segment words to look them up in the lexicon (for instance I think `'d`, `'ll`, and such are separate words in the English lexicon, and we should handle this even if we don't at the moment)